kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/model/VirtualCluster.java

```diff
@@ -229,6 +259,50 @@ private static NettyTrustProvider configureTrustProvider(Tls tlsConfiguration) {
         return new NettyTrustProvider(trustProvider);
     }
 
+    private static void configureCipherSuites(SslContextBuilder sslContextBuilder, Tls tlsConfiguration) {
+        Optional.ofNullable(tlsConfiguration.cipherSuites())
+                .map(ciphers -> sslContextBuilder.ciphers(
```

Comment:
nit:  you are relying on map having a side effect.  that's not how map is intended to be used.  `ifPresent` communicates the intent to the reader.

```suggestion
                .ifPresent(ciphers -> sslContextBuilder.ciphers(
```